ci/test.yml

```diff
@@ -269,20 +268,46 @@ connect-popup legacy:
     <<: *run_everything_rules
   when: manual
 
-connect-popup-webextension:
-  extends: .e2e connect-popup
+.e2e connect-popup-webextension:
+  stage: integration testing
```

Comment:
extending .e2e connect-popup does not work anymore? just asking, no big deal..

Reply:
I wanted to separate it to allow having different matrix with only methods like :

```
 parallel:
    matrix:
      - TEST_FILE: ["methods.test"]
```

Maybe there is a better way to reuse `.e2e connect-popup` and just overwrite the matrix.